### DISCUSSION TITLE
part-ii/2020-128/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#67] [Question comments]
Part II 2020 Quantum Information and Computation: Paper 2, Section II, 15C 

https://questions.tripos.org/part-ii/2020-128/

**Comment by thenewtonium** [comment]
**(c)** - in the ciruit diagram the ⦺ symbol denotes a $NOT$ gate.

<details> <summary><b>(d)</b> -  [sort of irrelevant rambling but also spoiler I guess]</summary>

The question is kind of silly since we have no prior on what $\ket{\psi}$ and $\ket{\phi}$ are, and a reasonable prior – uniform distribution – makes the probability that they are identical $0$.

The response that they wanted was essentially using that, if we repeat the process $N$ times, then $$\text{prob(we only ever measure 0's)} \to 0$$. But it doesn't do this uniformly, so for any fixed $N$ we can be fooled (with high probability) by states with $|\braket{\psi|\phi}|$ sufficiently close to 1...
</details>

Apparently (=according to Sergii) in 2020 QIC wasn't lectured by Josza, which explains some of the weirdness.
